virtualizarr/xarray.py

```diff
@@ -1,5 +1,14 @@
 from pathlib import Path
-from typing import List, Literal, Mapping, Optional, Union, overload, MutableMapping, Iterable
+from typing import (
+    List,
+    Literal,
+    Mapping,
+    Optional,
+    Union,
+    overload,
+    MutableMapping,
+    Iterable,
+)
```

Comment:
I think we're still not using `isort`, otherwise these would be in alphabetical order.

Reply:
On it!

Reply:
https://github.com/TomNicholas/VirtualiZarr/pull/98/commits/2c72571e5968a8390c40ed7fe4fd92a5ba27215e looks promising. Is this what you would like? There are also options to do this case sensitive?

Reply:
Just if anyone is curious. All I did was add the "I" class as active to the linter: https://github.com/TomNicholas/VirtualiZarr/pull/98/commits/8c128b57cdb67ae4447164b7c595696f2f3db8fb

Reply:
Cool! Thanks @jbusecke ❤️